app/views/entitlements/index.html.erb

```diff
@@ -5,52 +5,72 @@
 <div class="usa-section main-section">
   <h1><%= @page_title %></h1>
 
-  <p>Below you will find basic information and instructions regarding what you can ship.</p>
+  <aside class="sidenav">
+    <div class="sidenav-jump-to">
+      Jump to:
+    </div>
+    <ul class="usa-sidenav-list">
+      <li><%= link_to 'What am I allowed to bring?', '#can-bring' %></li>
+      <li><%= link_to 'Your Moving Allowance', '#moving-allowance' %></li>
+      <li><%= link_to 'Special Item Information', '#special-item-info' %></li>
+    </ul>
+  </aside>
+
+  <div class="sidenav-body">
+    <p>Below you will find basic information and instructions regarding what you can ship.</p>
+
+    <div class="what-to-expect-section">
```

Comment:
I'll rename this class in another PR since now it's used in more than just the "What to expect.." pages